packages/jss/src/types/jss.js

```diff
@@ -9,7 +9,7 @@ import SimpleRule from '../rules/SimpleRule'
 import FontFaceRule from '../rules/FontFaceRule'
 import type {CSSStyleRule} from './cssom'
 
-export type Classes = { [string]: string }
```

Comment:
same here, please check why we get these changes, back and forth

Reply:
Pretty sure that this was prettier because we set the bracket-spacing to false

Reply:
why wasn't it fixed before through lint-staged?

Reply:
I mean in previous releases

Reply:
I don't even see it with spaces in master

Reply:
https://github.com/cssinjs/jss/blob/master/packages/jss/src/types/jss.js#L12 I do🤔

Reply:
oh, sorry, npm run format fixed it for me locally and I didn't notice